agent/command/s3_put.go

```diff
@@ -85,10 +87,14 @@ type s3put struct {
 	// for missing files.
 	Optional string `mapstructure:"optional" plugin:"expand"`
 
+	// NoOverwrite, when set to true, will not upload files if they already exist in s3.
+	NoOverwrite string `mapstructure:"no_overwrite" plugin:"expand"`
```

Comment:
I think this naming convention is non-standard; can we call this SkipExisting instead?